### DISCUSSION TITLE
Account Dashboard Icon

### DISCUSSION THREAD

**Discussion by amyodea** [twilio-labs/paste#3881] [Request: Icon] [answered]
### Description

Hey Team! I need to add a new icon for 'Account Dashboard' into the Paste library. Thank you! 

![Screenshot 2024-04-29 at 10 33 49](https://github.com/twilio-labs/paste/assets/104831341/5a697777-27f3-4e10-b04d-55b0eb828a9a)

### Link to icon

https://www.streamlinehq.com/icons/streamline-light/business-products/trading/trading-website-browser

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by serifluous** [comment, marked as answer]
Closing this discussion to consolidate with #3880
